client/shipping-label/index.js

```diff
@@ -28,6 +28,7 @@ export default ( { formData, labelsData, paperSize, storeOptions } ) => ( {
 				labels: labelsData || [],
 				paperSize,
 				form: {
+					isSuccess: true,
```

Comment:
This is a convenience so the "Success" button is always shown when you open the dialog, no need to go through the flow. Delete this line after design.